src/seedu/addressbook/data/person/Email.java

```diff
@@ -24,7 +25,7 @@
     public Email(String email, boolean isPrivate) throws IllegalValueException {
```

Comment:
Some of these operations can be abstracted to the parent class. Please review how to best implement the commonly performed operations in child classes to the parent class.